FFXIVClientStructs/FFXIV/Component/SteamApi/SteamTypes.cs

```diff
@@ -23,4 +27,30 @@ public struct CSteamId {
         public uint AccountType => (byte)((RawSteamId >> 52) & 0xF);
         public uint AccountInstance => (uint)((RawSteamId >> 32) & 0xFFFFF);
     }
+    
+    /// <summary>
+    /// A structure that represents a Steam API context. This is used by <c>SteamInternal_ContextInit</c>
+    /// to manage and maintain a pointer to an instance of an arbitrary Steam API type.
+    /// </summary>
+    [StructLayout(LayoutKind.Explicit, Size = Size)]
+    public unsafe struct SteamInterfaceContext {
+        public const int Size = 0x18;
+        
+        [FieldOffset(0x0)] public delegate* unmanaged<void*, void*> FindOrCreateInterfaceFPtr;
+        [FieldOffset(0x8)] public nuint Counter;
+        [FieldOffset(0x10)] public void* CachedInterfacePtr;
+
+        /// <summary>
+        /// Gets a pointer to the Steam API instance stored in this context.
+        /// This pointer is suitable for use in imported Steam API methods.
+        /// </summary>
+        public nint GetInterface() {
+            fixed (SteamInterfaceContext* ptr = &this) {
+                // ToDo: Actually call `SteamInternal_ContextInit(ctx)` here, rather than just returning the cached pointer.
+                //       Needs some way to get the fptr for that method first. For now this is *probably fine* (and won't blow up)
+                //       but isn't awesome.
+                return (nint)ptr->CachedInterfacePtr;
```

Comment:
This would require some way for ClientStructs to be able to call a method from the import table by reference. I don't think this is particularly useful to have except here, but it may be a good thing to have in general.